Documentation/MigrationGuide.md

```diff
@@ -1,3 +1,15 @@
+
+# Migration Guides
+
+All the migration guides for SwiftGen are spread out over a few files, depending on the target audience.
+
+| Migration Guide | Target Audience | Abstract |
+|-----------------|-----------------|---------|
+| SwiftGen Migration Guide (this file) | All users of SwiftGen | Changes in configuration, CLI parameters and global changes overview |
+| [Templates Migration Guide](templates/MigrationGuide.md) | Users of SwiftGen's bundled templates (`templateName` option) | Some templates may have been renamed, removed or merged, or their functionality may have changed |
+| [SwiftGenKit Migration Guide](SwiftGenKit%20Contexts/MigrationGuide.md) | Template writers | Changes in names of variables provided by SwiftGenKit to your templates |
+| [StencilSwiftKit Migration Guide](https://github.com/SwiftGen/StencilSwiftKit/blob/master/Documentation/MigrationGuide.md) | Template writers | Changes in extra filters and tags for use in templates |
+
```

Comment:
Just for nitpicks: add a `---` separator line between that header and the rest?

Reply:
It looks really weird, I tried

Reply:
Well, locally, no idea how it looks on GH

Reply:
Ok, then (and stop reading my mind)

Let's merge then 🎉

Reply:
Okay, on GH it looks better

Reply:
Ah :wink: